fpage/templates/fpage/home.html

```diff
@@ -0,0 +1,203 @@
+{% block content %}
```

Comment:
Use template inheritance here and remove the duplicated code.

Reply:
I was using the template inheritance at first but the structure of the landing page was not coming properly so i shifted to this and i dont think we would be hardly making changes in the home page  so it wont create a problem

Reply:
It's not about creating problem. It's about following the DRY (Don't Repeat Yourself) in your code. It's a good practice while developing anything.

You can ask us(me and Mann) for help if you are facing problem regarding it but avoiding a good approaching to resolve a bug is not a good idea.

Reply:
It's not about creating problem. It's about following the DRY (Don't Repeat Yourself) in your code. It's a good practice while developing anything.

You can ask us(me and Mann) for help if you are facing problem regarding it but avoiding a good approaching to resolve a bug is not a good idea.

Reply:
Ok will follow the DRY method. I will change it and make a commit